ivy/functional/ivy/manipulation.py

```diff
@@ -707,7 +711,7 @@ def squeeze(
         b: ivy.array([3., 4., 5.])
     }
     """
-    return current_backend(x).squeeze(x, axis, out=out)
+    return current_backend(x).squeeze(x, axis,copy=copy, out=out)
```

Comment:
Hi! There is a lint error here: `ivy/functional/ivy/manipulation.py:714:46: E231 missing whitespace after ','`
Could you quickly fix that please?

Reply:
Done!